srml/support/procedural/src/storage/parse.rs

```diff
@@ -113,11 +113,16 @@ struct DeclStorageLine {
 	pub default_value: ext::Opt<DeclStorageDefault>,
 }
 
+#[derive(Parse, ToTokens, Debug)]
+struct DeclStorageGetterBody {
```

Comment:
This is not required. You can just add the `fn_token: Option<syn::token::Fn>` to the getter struct.